### PR TITLE
Allow `autoStart` param to be overridden

### LINKED ISSUE #5: Support overriding more props
For example, it would be useful to set `autoStart` to `false`

### PR DESCRIPTION
**Pull request by milesblackwood**
Also provides a default value. This would be an example of an additional param as mentioned in #5

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -59,7 +60,7 @@ export default class Brightcove extends React.Component {
           <param name="isUI" value="true" />
           <param name="isVid" value="true" />
           <param name="dynamicStreaming" value="true" />
-          <param name="autoStart" value="true" />
+          <param name="autoStart" value={autoStart} />
```
Comment (keithamus) [inline src/index.js]: Should this be `{String(autoStart)}` as the value brighcove expects is actually a String of `"false"` or `"true"`?

Comment (milesblackwood) [inline src/index.js]: Good catch!

**Comment by keithamus** [comment]
LGTM. @gcedo or @danielmoll want to take a look?

**Comment by gcedo** [comment]
LGTM

**Pull request merged**